lab_6_pipeline/pos_frequency_pipeline.py

```diff
@@ -2,7 +2,10 @@
 Implementation of POSFrequencyPipeline for score ten only.
 """
 from typing import Optional
+from pathlib import Path
 
+from core_utils.article.article import Article
```

Comment:
why do you touch them when doing work for 6?

Reply:
done